freshmaker/errata.py

```diff
@@ -514,3 +517,26 @@ def _get_release(self, errata_id):
     def is_zstream(self, errata_id):
         release = self._get_release(errata_id)
         return release["data"]["attributes"]["type"] == "Zstream"
+
+    def has_jira_major_incidents(self, errata_id: str) -> bool:
+        """
+        Checks if this errata has a 'major incident' issue in JIRA
+
+        :param errata_id: The ID of the errata advisory
+        :type errata_id: str
+
+        :return: Wether the errata has or not a major incident issue in JIRA
+        :rtype: bool
+        """
+        resp = self._get_advisory(errata_id=errata_id)
+        mi_pattern = re.compile(r"major\s*incident", re.IGNORECASE)
+
+        if resp.get("jira_issues", False):
+            for issue in resp["jira_issues"]:
```

Comment:
This doesn't work, the data structure in response is more complex than this, `resp["jira_issues"]` is a dict, you have to use `resp.get("jira_issues", {}).get("jira_issues", {})` to get list of jira issues, I'd suggest you to use Errata's `/jira_issues/<advisory>.json` API to get jira issues, which is more friendly.

You can refer to CWFHEALTH-2362 and the linked MR.

Reply:
Done! That's indeed a much friendlier endpoint, thanks a lot @qixiang! 😄